src/lib.rs

```diff
@@ -47,9 +47,16 @@ pub mod mutex;
 pub mod proxy;
 
 pub use mutex::BusMutex;
+pub use mutex::NullMutex;
 pub use proxy::BusManager;
 pub use proxy::BusProxy;
 
+/// Type alias for a bus manager using the [`shared_bus::NullMutex`].
+///
+/// This bus manager can be used when all bus users are contained in a single execution context,
+/// i.e. no synchronization between different tasks/threads is needed.
+pub type NullBusManager<L, P> = BusManager<NullMutex<L>, P>;
```

Comment:
Maybe call this a `SingleThreadedBusManager` or `SingleContextBusManager` to make it clear that this can only be used when operating in a single context?

Reply:
Yeah, `NullBusManager` didn't sound good to me either.  Will change it!